test/fixtures/broken-pipe/basic.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+global.process = { __proto__: process, pid: 123456 }
+Date.now = function () { return 1459875739796 }
+require('os').hostname = function () { return 'abcdefghijklmnopqr' }
+
+const pino = require('../../..')()
```

Comment:
How is this test different than the `destination.js` test? Isn't the default value `pino.destination()`?

Reply:
This uses a different code path. We were creating SonicBoom instances on several places.